swagger-apis/payments/2.1.0.yml

```diff
@@ -0,0 +1,2596 @@
+        status:
+          $ref: '#/components/schemas/EnumPaymentStatusType'
+        rejectionReason:
+          $ref: '#/components/schemas/RejectionReason'
+        localInstrument:
+          $ref: '#/components/schemas/EnumLocalInstrument'
+        cnpjInitiator:
+          type: string
+          pattern: '^\d{14}$'
+          maxLength: 14
+          example: '50685362000135'
+          description: CNPJ do Iniciador de Pagamento devidamente habilitado para a prestação de Serviço de Iniciação no Pix.
+        payment:
+          $ref: '#/components/schemas/PaymentPix'
+        transactionIdentification:
+          type: string
+          pattern: '^[a-zA-Z0-9]{1,35}$'
+          maxLength: 35
+          example: E00038166201907261559y6j6
+          description: |
+            Trata-se de um identificador de transação que deve ser retransmitido intacto pelo PSP do pagador ao gerar a ordem de pagamento.
+
+            [Restrição] A detentora deve obrigatoriamente retornar o campo com o mesmo valor recebido da iniciadora, caso ele tenha sido enviado na requisição da iniciação do pagamento.
+        remittanceInformation:
+          type: string
+          maxLength: 140
+          pattern: '[\w\W\s]*'
+          example: Pagamento da nota RSTO035-002.
+          description: |
+            Deve ser preenchido sempre que o usuário pagador inserir alguma informação adicional em um pagamento, a ser enviada ao recebedor.
+        creditorAccount:
+          $ref: '#/components/schemas/CreditorAccount'
+        cancellation:
+          $ref: '#/components/schemas/PixPaymentCancellation'
+        debtorAccount:
+          $ref: '#/components/schemas/DebtorAccount'
+    Schedule:
+      type: object
+      description: |
+        [Restrição] Mutuamente excludente com o campo date. 
+
+        Este campo é obrigatório no caso de agendamento. 
+
+        Neste caso, o campo date não deve ser informado.
+      required:
+        - single
+      properties:
+        single:
+          type: object
+          description: Define a política de agendamento único.
+          required:
+            - date
+          properties:
+            date:
+              type: string
+              format: date
+              maxLength: 10
+              pattern: '^(\d{4})-(1[0-2]|0?[1-9])-(3[01]|[12][0-9]|0?[1-9])$'
+              example: '2021-01-01'
+              description: |
+                Define a data alvo da liquidação do pagamento. 
+
+                O fuso horário de Brasília deve ser utilizado para criação e racionalização sobre os dados deste campo.
+
+                Observação: Esse campo deverá sempre ser no mínimo D+1 corrido, ou seja, a data imediatamente posterior em relação a data do consentimento considerando o fuso horário de Brasília e deverá ser no máximo D+365 corridos a partir da data do consentimento considerando o fuso horário de Brasília
+    XFapiInteractionId:
```

Comment:
This should be required?

Reply:
We have two XFapiInteractionId.

1. It's a component into structure parameter/XFapiInteractionId which responsable just for parameter of endpoints . **It componet is required.**
2. It's a component into structure scheme/XFapiInteractionId which shared with others componentes like InternalServerError

Reply:
Gotcha, so all good?